app/js/app/directives/equation_editor/MenuSymbol.js

```diff
@@ -82,37 +93,63 @@ define([], function() {
 
                     element.css("left", 0);
                     element.css("top", 0);
-                    
-                    scope.$emit("symbolDrop", scope.symbol, pageX, pageY, pageX, pageY);
+
+                    // This ensures new symbols can be selected.
```

Comment:
What does the below code do, and why? What are the debugs for, and if they're staying - why the odd numbering?